app/src/components/governance/ParticipantList.vue

```diff
@@ -78,10 +89,6 @@ export default {
     },
   },
   methods: {
-    getParticipantName(participant) {
-      const name = participant.name
-      return name.length > 25 ? formatAddress(name) : name || `n/a`
-    },
```

Comment:
It is going to be too long, no?

Reply:
Look, this is how it looks on mobile:

![image](https://user-images.githubusercontent.com/40721795/93495928-f1cfb080-f90e-11ea-8d7f-23c581dc2e65.png)

Reply:
Ok, we'll do it in another PR

Reply:
we'll leave this now. in the API if there is no name, it should be null, and the front end will display + format the address when there is no name.